mender-dist-packages-building/armhf/buster_add_cmake.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+# v3.21 is the oldest version that we can build from source in QEMU armhf. See:
+# https://gitlab.kitware.com/cmake/cmake/-/issues/22328
+CMAKE_VERSION="v3.21.6"
+_CMAKE_VERSION=$(echo "$CMAKE_VERSION" | sed 's/v//')
+
+# First remove OS cmake
+apt-get remove --assume-yes cmake
+apt-get autoremove --assume-yes
```

Comment:
Are you sure `autoremove` is a good idea? Doesn't that also remove dependencies that cmake has? Which might be needed by the source build also?

Reply:
Based on my experiments, building Cmake from source builds all the dependencies and these can be removed. Anyway I also think it is safer to leave libraries around so I will instead do `remove cmake cmake-data` and leave the rest be.

By the way, strictly speaking we don't need even to remove `cmake` debian package, as building from source the newer CMake gets intalled in `/usr/local/bin/cmake` and `/usr/local/share/cmake-3.21/`. I thought about removing this to avoid potential miss-configurations.